tests/unittest_mlagent/unittest_mlagent.cc

```diff
@@ -46,9 +46,40 @@ _init (const std::vector<MockModel> &models = default_models)
   }
 }
 
+constexpr gchar valid_activated_uri_format_literal[] = "mlagent://model/%s";
 constexpr gchar valid_uri_format_literal[] = "mlagent://model/%s/%u";
 constexpr gchar invalid_uri_format_literal[] = "ml-agent://model/%s/%u";
 
+/**
+ * @brief tests of getting activated model path with valid URIs
+ */
+TEST (testMLAgent, GetActivatedModelValidURIs_p)
+{
+  _init ();
```

Comment:
How about using Test fixture and use `SetUp` function instead of `_init`? (later in another PR)

Reply:
Cool. I'll use `SetUp` in later PRs